mail_footer_notified_partners/models/mail_followers.py

```diff
@@ -48,16 +48,17 @@ def get_signature_footer(
 
         return footer
 
-    @api.model
+    @api.multi
```

Comment:
I repeat: this cannot be multi or one because `message_id` is from a different model than `mail.notification`